crates/engine/tree/src/tree/payload_processor/prewarm.rs

```diff
@@ -27,6 +28,9 @@ use std::{
 };
 use tracing::{debug, trace};
 
+/// Optimism deposit transaction type (0x7E/126).
+const OPTIMISM_DEPOSIT_TX_TYPE: u8 = 126;
+
```

Comment:
pls remove in favour of importing `DEPOSIT_TX_TYPE_ID` from op-alloy

Reply:
ah perhaps it makes sense to have this double up for now, since we probably don't want to have op-alloy be a dep to any crate outside of the optimism directory...